internal/pkg/fleetapi/action.go

```diff
@@ -191,7 +191,7 @@ func (a *ActionPolicyChange) Expiration() (time.Time, error) {
 
 // ActionUpgrade is a request for agent to upgrade.
 type ActionUpgrade struct {
-	ActionID         string `yaml:"action_id"`
+	ActionID         string `yaml:"id"`
```

Comment:
@michel-laterman I had to change the yaml serialization field name back to what it was before in order to fix the issue with .update-market. Please double check that has no other side-effects in the context of the original changes mentioned in the description of this PR.